tripal_chado/src/Plugin/TripalBackendPublish/ChadoPublish.php

```diff
@@ -1081,7 +1095,7 @@ public function publish_init(array $options) : bool {
     }
     // Optional values
     $this->schema_name = $options['schema_name'] ?? 'chado';
-    $this->republish = ($options['republish'] ?? FALSE) ? TRUE : FALSE;
+    $this->republish = $options['republish'] ?? TRUE;
```

Comment:
`$this->republish` is declared as a boolean, while `$options['republish']` will contain a `1` or `0` so this odd construct was used to convert to boolean. Maybe safer would be to use `$this->republish = boolval($options['republish'] ?? 1)`

Reply:
Solved in commit 77e068788b610974730e4dff44ce06f29a563953 as you suggested.